packages/sql-mapper/lib/entity.js

```diff
@@ -200,6 +200,7 @@ function createMapper (defaultDb, sql, log, table, fields, primaryKeys, relation
     lt: '<',
     lte: '<=',
     like: 'LIKE',
+    ilike: 'ILIKE',
```

Comment:
add a `hasILIKE` property to the queries objects https://github.com/platformatic/platformatic/tree/34af62a4ef68e6bdddf5693e042b689cd2930b9f/packages/sql-mapper/lib/queries, and then do `ilike: queries.hasILIKE ? 'ILIKE' : 'LIKE'`.

It's likely our best bet